__tests__/db.test.js

```diff
@@ -0,0 +1,83 @@
+const data = require("../db/data/test-data")
+const app = require("../app")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const request = require("supertest")
+
+beforeEach(() => seed(data))
+
+afterAll(() => {
+  db.end()
+})
```

Comment:
your afterAlls are asynchronous, so you're going to need a return in here - otherwise jest won't exit successfully 

Or you could put your db.end() in one line like you did for your seed function